contract/src/pegasus.js

```diff
@@ -271,11 +187,23 @@ const makeCourier = ({
     const localAmount = packetToLocalAmount(packet);
     const depositFacet = await E(board).getValue(depositAddress);
```

Comment:
I see why we're using the board here. But this means we can only deposit into a depositFacet that's publicly available in the board. We should eventually find a way for AliceG to transfer to AliceA without forcing AliceA to accept gifts from everyone.

I suppose since `board` is a parameter of the contract, AliceA could provide a board of her own construction.

Reply:
Currently, the wallet's depositFacet (which is on the board) includes a confirmation action for inbound payments.  If you want to skip the confirmation action, you can mark an individual purse with an "autodeposit" flag in the wallet UI.